doc-content/enterprise-only/pages/building-custom-dashboard-widgets-creating-all-processes-heatmap-components-proc.adoc

```diff
@@ -61,7 +61,7 @@ You can modify the SQL query according to your database if required.
 The new page opens in the *Page Editor* and you can see that the *Heatmaps* component is available under the *Components* panel.
 . In the *Components* panel, expand the *Heatmaps* components and drag the *All Processes Heatmaps* component type to the canvas.
 . In the *Displayer editor* wizard, click *Data* tab and select the newly created {KIE_SERVER} dataset.
-. In the *Data* tab, select *EXTERNALID*, *PROCESSID*, *NID*, and *AVERAGEEXECUTIONTIME* from the *Columns* field.
+. In the *Data* tab, select *NODEID* and *TOTAL_HITS* from the *Columns* field.
```

Comment:
Hi @kaldesai ,
for all process heapmap column values are EXTERNALID,PROCESSID,NODEID and TOTAL_HITS